sos/plugins/samba.py

```diff
@@ -33,6 +33,10 @@ def setup(self):
             "/var/log/samba/log.winbindd-dc-connet",
             "/var/log/samba/log.wb-*"
         ])
+
+        if self.get_option("all_logs"):
+            self.add_copy_spec("/var/log/samba/")
+
```

Comment:
As I said, it is preferred to handle them separately (if nothing else, it is consistent with the main `logs` plugin that handles `syslog` and `journald` logs).

I'm happy to fix up the `add_copy_spec_limit()` API warts if you like; they've been on my todo list for some time (really, the separate `..._limi()` version should die, and the regular `add_copy_spec()` would get a new `sizelimit` kwarg).